src/index.js

```diff
@@ -166,6 +168,18 @@ class Backendless {
     return app
   }
 
+  appInfoPromise() {
```

Comment:
the rare case but it could be:

the `initApp` might be called several times with a different credential, so we need to have some kind of `reset appInfoPromise` 

```
appInfoPromise(reset) {
if (reset || !this.__appInfoPromise) {
....
}
```